### PR TITLE
Ignore bad pylint warning

### PR DESCRIPTION
**Pull request by anonymoose2**
## Proposed changes
- Ignore bad pylint warning

## Brief description of rationale
This is causing CI to fail.

### CODE REVIEW

File: intranet/apps/eighth/serializers.py [hunk 1]
```diff
@@ -93,7 +93,13 @@ class EighthBlockDetailSerializer(serializers.Serializer):
     comments = serializers.CharField(max_length=100)
 
     def process_scheduled_activity(
-        self, scheduled_activity, request=None, user=None, favorited_activities=None, recommended_activities=None, available_restricted_acts=None
+        self,
+        scheduled_activity,
+        request=None,
+        user=None,
+        favorited_activities=None,
+        recommended_activities=None,  # pylint: disable=unused-argument
```
Comment (theo-o) [inline intranet/apps/eighth/serializers.py:101]: Isn't this argument unused?

Comment (theo-o) [reply intranet/apps/eighth/serializers.py:101]: I removed the only use in ea97f5c0f4ac3b4d1fefb1da8dadc983a04159cc

Comment (anonymoose2) [reply intranet/apps/eighth/serializers.py:101]: Oops, that's actually correct. I was looking at an older version when I made the change.

I'll try broader changes to remove the variable further up.

**Comment by anonymoose2** [comment]
Closing in favor of #1012 .

**Pull request closed without merging**